clox/functions.py

```diff
@@ -262,6 +264,7 @@ def main() -> None:
     parser.add_argument('--hide-timezone', help='hide timezone', nargs="?", const=1)
     parser.add_argument('--am-pm', help='AM/PM mode', nargs="?", const=1)
     parser.add_argument('--calendar', help='calendar mode', type=str.lower, choices=CALENDARS_LIST)
+    parser.add_argument('--first-weekday', help='first weekday', type=int, default=0)
```

Comment:
I believe it would be better to accept the `--first-weekday` argument in a string format, for example:

```console
clox --calendar="month" --first-weekday="monday"
```

Reply:
It's best to support both `int` and `str` types: `int` for originality and `str` for user-friendliness.

Reply:
based on what we discussed I will implement only the name (full and abrivetated form)